### DISCUSSION TITLE
"Sign in with Passkey" Integration in Firebase Auth

### DISCUSSION THREAD

**Discussion by skavinvarnan** [firebase/flutterfire#11398] [Feature request]
As Passkey is steadily gaining traction and has already gained official support from [Google Identity](https://developers.google.com/identity/passkeys), it would be highly beneficial to users if we could incorporate "Sign in with Passkey" as an official integration for Firebase Auth.

**Comment by cbenhagen** [comment]
Relevant links:
https://firebase.uservoice.com/forums/948424-general/suggestions/46647016-support-authentication-with-passkeys
https://issuetracker.google.com/issues/289237739

Please star the issue and hit the +1 button but comment only if you have more information to add. Thanks ;)

**Comment by utamori** [comment]
Finally coming !
https://firebase.uservoice.com/forums/948424-general/suggestions/46647016-support-authentication-with-passkeys

**Comment by cbenhagen** [comment]
Issue for the Passkey preview support: https://github.com/firebase/flutterfire/issues/17201

**Comment by MauScheff** [comment]
Recently, "Sign in with Apple" stopped working because of increasing Apple's privacy protections, se we had to move to using sign in with redirect instead of sign in with popup, for this we had to self-host, and figure a bunch of things out. Recently, I discovered, that under this configuration, when someone taps on "Sign in with Google", then the iPhone prompts if it want's them to use the Passkey (eg. just show their face and not enter their email), then they are redirected back to the login screen, but without signing in and no errors or anything. Is this related with the lack of "Sign in with Passkey" support in FirebaseUI? I understand the goal of that product is that developers don't have to roll their own Auth flows which is time-consuming and error-prone and it supports all popular providers out-of-the-box. On these lines, I would think that it's critical that users be able to "Sign in with Passkey" for all developers who rely on FirebaseAuth, and benefit from FirebaseUI.

**Comment by Yashasewi** [comment]
is there any update ..
